modules/ldk/ldk_lib/src/lib.rs

```diff
@@ -24,9 +30,65 @@ pub unsafe extern "C" fn ldk_des_invoice(input: *const std::os::raw::c_char) ->
             };
             let is_currency_bitcoin = invoice.currency() == Currency::Bitcoin;
             let has_payment_hash = invoice.payment_hash().is_some();
+            if has_payment_hash && is_currency_bitcoin && is_signature_valid {
+                let mut result = String::new();
+
+                result.push_str("HASH=");
+                result.push_str(&invoice.payment_hash().unwrap().0.to_string());
+
+                result.push_str(";AMOUNT=");
+                if let Some(amount) = invoice.amount_pico_btc()  {
+                    let amount_milisatoshis = amount/10;
```

Comment:
If you need the milisatoshis, you could use `amount_milli_satoshis` instead of `amount_pico_btc`. Otherwise, I think this `amount/10` could bring some issues.